src/components/RisCopyableLabel/RisCopyableLabel.stories.ts

```diff
@@ -0,0 +1,47 @@
+import { html } from "@/lib/tags";
+import { Meta, StoryObj } from "@storybook/vue3";
+import RisCopyableLabel from ".";
+
+const meta: Meta<typeof RisCopyableLabel> = {
+  component: RisCopyableLabel,
+
+  tags: ["autodocs"],
+
+  args: {
+    text: "Copy to clipboard",
+    value: undefined,
+    name: undefined,
+  },
+
+  argTypes: {
+    value: String,
+    name: String,
+  },
+};
+
+export default meta;
+
+export const Default: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { RisCopyableLabel },
+    setup() {
+      return { args };
+    },
+    template: html`<RisCopyableLabel v-bind="args" />`,
+  }),
+};
+
+export const CustomValue: StoryObj<typeof meta> = {
+  args: {
+    text: 'Copy "test" to clipboard',
+    value: "test",
+  },
+
+  render: (args) => ({
+    components: { RisCopyableLabel },
+    setup() {
+      return { args };
+    },
+    template: html`<RisCopyableLabel v-bind="args" />`,
```

Comment:
This does not render any code in my local Storybook for some reason

Reply:
Good catch! Seems like in this:

```
text: 'Copy "test" to clipboard',
```

... the `"` inside the string break Storybook's code rendering. Changed it to single quotes, now it should work.